installer/PowerToysSetup/Product.wxs

```diff
@@ -1356,9 +1356,9 @@
             </Component>
 
 			<!-- !Warning! Make sure to change Component Guid if you update the file list -->
-            <Component Id="launcherInstallComponent" Guid="5E7074E9-155F-49A1-BD4C-470A09814618" Directory="LauncherInstallFolder" >
+            <Component Id="launcherInstallComponent" Guid="C3161BDF-C5F9-47C3-863A-E326D04EEC28" Directory="LauncherInstallFolder" >
```

Comment:
@yuyoyuppe Is guid change ok here?

Reply:
@crutkas I changed the guid because of the comment above the Wix component stated to do so.

Reply:
great call out, clearly i can't read.  hahha

Reply:
Me neither 🤷‍♂️